nix/check-stylish-network.nix

```diff
@@ -15,5 +15,15 @@ runCommand "check-stylish-network" {
   fd -p network-mux -e hs -E Setup.hs -E network-mux/src/Network/Mux/Bearer/Pipe.hs -E network-mux/src/Network/Mux/Channel.hs -X stylish-haskell -c .stylish-haskell-network.yaml -i
   fd -p ouroboros-network -e hs -E Setup.hs -X stylish-haskell -c .stylish-haskell-network.yaml -i
```

Comment:
In #3719 I noticed that the above does matches all the files if the parent directory is also called `ouroboros-network`.  It should be done as in this [patch](https://github.com/input-output-hk/ouroboros-network/pull/3719/commits/3167ff4bb12113f7aa63f52edb85bd7ae7c20cc9).  I think you can just cherry-pick it.

Reply:
Cherries picked! :smiley: